src/go/rpk/pkg/cli/cmd/cluster/health.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// Package brokers contains commands to talk to the Redpanda's admin brokers
+// endpoints.
+package cluster
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewHealthOverviewCommand(fs afero.Fs) *cobra.Command {
+	var (
+		wait bool
+		exit bool
+	)
+	cmd := cobra.Command{
+		Use:   "health",
+		Short: "Queries cluster for health overview.",
```

Comment:
It may be beneficial to have a bit of long text that describes the cases what it means for a cluster to be healthy or not healthy.